copier.yml

```
@@ -11,6 +11,11 @@ description:
     type: str
     help: What is the description of this repository?
 
+install_claude_cli:
+    type: bool
+    help: Should the Claude CLI be installed in the devcontainer?
+    default: no
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)